openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/method/CustomMethod.scala

```diff
@@ -16,6 +16,7 @@ import reactor.core.scala.publisher.SMono
 object CapabilityIdentifier {
   val LINAGORA_ECHO: CapabilityIdentifier = "com:linagora:params:jmap:echo"
   val LINAGORA_FILTER: CapabilityIdentifier = "com:linagora:params:jmap:filter"
+  val LINAGORA_PGP: CapabilityIdentifier = "com:linagora:params:jmap:pgp"
```

Comment:
Can we have tests showing this is advertised in the session?

Reply:
Can we have tests were we call the session endpoint and ensure the capability is well exposed in the session?

I did not see them....